flow/project.py

```diff
@@ -104,9 +104,18 @@ class _condition(object):
     current_arbitrary_tag = 0
 
     def __init__(self, condition, tag=None):
-        # Add tag to differentiate built-in conditions during graph detection.
+        '''Add tag to differentiate built-in conditions during graph detection.
```

Comment:
I think we use [triple-double](https://en.wikipedia.org/wiki/Double-double_(basketball)#Triple-double) quotes `"""` everywhere else. Triple single quotes are harder to read.
```suggestion
        """Add tag to differentiate built-in conditions during graph detection.
```